cypress.json

```diff
@@ -0,0 +1,6 @@
+{
+  "baseUrl": "http://localhost:3000",
```

Comment:
Do you have a plan to run the test locally in parallel with development mode? 🙌
Or this could work with `headless` option? 🤔

Reply:
Thank you, I should change this value.
Fixed in https://github.com/nimblehq/til-web/pull/17/commits/d471ccaee11d5af40dc7f17fc49ff93902d56916

Reply:
@hoangmirs It might fail the test, you have to pass port in the git-action also